src/Test/TestAIConstraints.java

```diff
@@ -82,4 +86,30 @@ private boolean testUnmovedFriend(AIMaker ai)
 
     return testPassed;
   }
+
+  private boolean testBuildTooManyMans(AIMaker ai)
+  {
+    setupTest(TestRange.getMapInfo(), ai, new GameScenario(new AWBWUnits(), 1000, 1000, 1, FogMode.OFF_DOR, TagMode.OFF));
```

Comment:
Probably make a temp variable for the altered unit cap, since that's the point of the test.